src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+
+type Props = {
+  error: string;
+  setError: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const ErrorMsg: React.FC<Props> = ({ error, setError }) => {
+  useEffect(() => {
+    if (error === '') {
```

Comment:
```suggestion
    if (!error) {
```

empty string is falsy value